### DISCUSSION TITLE
Developers Meetings

### DISCUSSION THREAD

**Discussion by mxochicale** [SciKit-Surgery/sustainable-pkg-stats#35] [General]
Hi all, 

Feel free to add notes and minutes from developers' meetings.

Thanks

**Comment by yidilozdemir** [comment]
Meeting notes for 11/05/2023

to-do:
- check out get_loc.py script, how it is run and integrated to the github pages workflow/makefile
- cloc library 
- libraries locally shouldn't (ideally) be saved, and cacheing might be good to not fetch information/process again-- the main concern here is Github API might have a call limit, so this needs to be checked
- cookiecutter- look into the scikit-surgery cookiecutter template
- github actions start

What to do next (priority list)
1. Github Actions start, for now just use the makefile (so the links leading to other links is todo)
2. get_loc script, run and where to integrate it 
3. Libraries/ etc and general data saving pruning 

Idil's note: 3 might go hand in hand with 1 because if the process is more lightweight (no folders saved etc), it might be easier to implement it to GH

**Reply by yidilozdemir** [reply]
For github actions, first trigger to create from a normal (push/PR) then with cron on a scheduled basis
